src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -92,13 +92,14 @@ def load_arguments(self, _):
         c.argument('src_name', help='The name of the source App Configuration.')
         c.argument('src_connection_string', validator=validate_connection_string, help="Combination of access key and endpoint of the source store.")
         c.argument('src_key', help='If no key specified, import all keys by default. Support star sign as filters, for instance abc* means keys with abc as prefix. Similarly, *abc and *abc* are also supported. Key filtering not applicable for feature flags. By default, all feature flags with specified label will be imported.')
-        c.argument('src_label', help="Only keys with this label in source AppConfig will be imported. If no label specified, import keys with null label by default.")
+        c.argument('src_label', help="Only keys with this label in source AppConfig will be imported. If no src-label specified, import keys with null label by default. Support star sign as filters, for instance * means all labels, abc* means labels with abc as prefix. Similarly, *abc and *abc* are also supported.")
+        c.argument('preserve_labels', arg_type=get_three_state_flag(), help="Original labels from source AppConfig will be preserved for all KVs. This argument should only be specified when --label is NOT provided.")
```

Comment:
>Original labels from source AppConfig will be preserved for all KVs.  [](start = 77, length = 69)

nit: Flag to preserve labels from source App Configuration. This argument should be specified along with --label.